step-api/step-api-keyword/src/main/java/step/handlers/javahandler/KeywordExecutor.java

```diff
@@ -185,8 +185,12 @@ private Output<JsonObject> invokeMethod(Method m, Input<JsonObject> input, Abstr
 			script.setProperties(properties);
 			script.setOutputBuilder(outputBuilder);
 
+			Keyword annotation = m.getAnnotation(Keyword.class);
+			String keywordName = annotation.name().isEmpty()?m.getName():annotation.name();
```

Comment:
@jeromebrongniart using input.getFunction() would be better (no need to duplicate the fallback logic in case of an empty name field)

Reply:
Done: https://github.com/exense/step-api/pull/18/commits/c6a1289c25afb069166fb7af24381df1287112c7